testdata/ostrusted-inferrable-test/Constants.java

```diff
@@ -0,0 +1,9 @@
+public interface Constants {
```

Comment:
It seems `testdata/ostrusted-inferrable-test` is not a proper place to put this case in. Where should it be put?